### PR TITLE
Create 10-Intro-to-Causal-AI

### PR DESCRIPTION
**Pull request by emrekiciman**
Adding a link to Paul and Jermain's recent Datacamp tutorial on causality using DoWhy

### CODE REVIEW

File: _case_studies/10-Intro-to-Causal-AI [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Intro to Causal AI Using the DoWhy Library in Python
+layout: page
+description: >-
+  This tutorial provides an introduction to causal AI using the DoWhy library in Python. It discusses fundamental principles and offers code examples.
+summary: >-
+  This tutorial covers some of the fundamental concepts and ideas of causal AI using the DoWhy library in Python. Causal inference is quite different 
+conceptually from standard machine learning, so most people will start out with limited background knowledge. However, basic knowledge of regression
```
Comment (fverac) [inline _case_studies/10-Intro-to-Causal-AI]: Needs an indent to render properly
```suggestion
  conceptually from standard machine learning, so most people will start out with limited background knowledge. However, basic knowledge of regression
```

File: _case_studies/10-Intro-to-Causal-AI [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+analysis and statistical modeling will be helpful for following along. By Paul Hünermund and Jermain Kaminski
```
Comment (fverac) [inline _case_studies/10-Intro-to-Causal-AI]: same as above
```suggestion
  analysis and statistical modeling will be helpful for following along. By Paul Hünermund and Jermain Kaminski
```

File: _case_studies/10-Intro-to-Causal-AI
Comment (fverac) [inline _case_studies/10-Intro-to-Causal-AI:1]: I would consider adding a .md file extension to this to be consistent with the other files in the folder. It still works without the .md though

Comment (emrekiciman) [reply _case_studies/10-Intro-to-Causal-AI:1]: Done!

File: _case_studies/10-Intro-to-Causal-AI (hunk 1 above)
Comment (emrekiciman) [reply _case_studies/10-Intro-to-Causal-AI]: added indentation

File: _case_studies/10-Intro-to-Causal-AI (hunk 2 above)
Comment (emrekiciman) [reply _case_studies/10-Intro-to-Causal-AI]: added indentation

**Comment by amit-sharma** [comment]
Looks good. thanks for adding.

**Review by fverac — APPROVED** [review]

**Pull request merged**